server/sonar-main/src/main/java/org/sonar/application/es/EsInstallation.java

```diff
@@ -33,14 +35,20 @@
  * This class does not ensure files nor directories actually exist.
  * </p>
  */
-public class EsFileSystem {
+public class EsInstallation {
```

Comment:
We chose the name "EsInstallation", because it now includes both paths and settings.